### DISCUSSION TITLE
Keyboard navigation scrolling on laptop

### DISCUSSION THREAD

**Discussion by keybasegit** [hbstack/.github#1] [Issues and bugs]
On arch Linux, Firefox, Brave, Librewolf browser I can scroll the page using a mouse. Page doesn't respond to keyboard keys up and down.

**Comment by razonyang** [comment]
Thanks for the feedback, fixed by https://github.com/hugomods/search/pull/116. Should be fixed after upgrading to `v0.1.2`:

```sh
hugo mod get -u github.com/hugomods/search@v0.1.2
```

**Comment by keybasegit** [comment]
Solved @Vlad 😁
